### ISSUE TITLE
Kubernetes operator OC_RESOURCE_LABELS support

### ISSUE THREAD

**Issue by mwuertinger** [census-ecosystem/opencensus-go-exporter-stackdriver#118]
The [OpenCensus Kubernetes Operator](https://github.com/census-ecosystem/kubernetes-operator) sets the environment variables `OC_RESOURCE_TYPE` and `OC_RESOURCE_LABELS`. The function `monitoredresource.Autodetect()` should use those values if present.

**Comment by songy23** [comment]
Stackdriver exporter will be migrated to use the Resource APIs (https://github.com/census-instrumentation/opencensus-go/tree/master/resource) once they're released. Resource APIs support reading the resource types and labels.

**Comment by mwuertinger** [comment]
Does this mean that `monitoredresource.Autodetect()` will be deprecated then?

**Comment by songy23** [comment]
Yes, `monitoredresource` will be deprecated and generic `resource` will be used instead, similar to what happened in Java: https://github.com/census-instrumentation/opencensus-java/pull/1744 and https://github.com/census-instrumentation/opencensus-java/pull/1745.
